tiny_hippo_run.py

```diff
@@ -43,10 +45,13 @@ def main():
     for rule in rules_system_privacy:
         rule()
 
-    # Note: Steps 2 and 3 happen simultaneously in the "sniff()" call, but are separated for clarity
-    # 2) Capture IoT packets only with crafted sniff
+    # 3) Perform a scanning analysis of the IoT devices
+    ip_to_mac = __pair_ip_to_mac()
+    for rule in rules_scanning_privacy:
+        rule(ip_to_mac)
```

Comment:
I know time was a concern for how long these scans would be taking. What ended up being the final total time per IP for these things? Or was that the other NMAP scanning?

Reply:
Not bad, on minute on startup won't be too much of an issue imo.